docs/static-pvc.md

```diff
@@ -269,5 +279,42 @@ $ kubectl create -f cephfs-static-pvc.yaml
 persistentvolumeclaim/cephfs-static-pvc created
 ```
 
-**Note** deleting PV and PVC does not delete the backend CephFS subvolume,
+### Verify PVC creation
```

Comment:
can you please update the rbd example with the verification step as well?

Reply:
Sure, but in our scenario, we use OpenEBS to provide LocalPV for database service persistence, so I cannot provide a solution immediately. 

I will test it in the cluster and provide a verification method suitable for RBD🫡.